server/app/models/va/SocialMatch.js

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ * This model corresponds to the SocialMatch neo4j nodes
```

Comment:
Just a note for the future: everything in this /models/va/ directory is only useful in the context of the neode ogm. As much as possible, we are trying to move to a more raw cypher database interaction, reducing our use of neode.

Reply:
for specifying the models it seems a lot safer to keep neode, but minimize its use aside from that.